sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.institutions.controller.getCompletedApplications.e2e-spec.ts

```diff
@@ -0,0 +1,260 @@
+  saveFakeApplicationOfferingRequestChange,
+} from "@sims/test-utils";
+import {
+  ApplicationOfferingChangeRequestStatus,
+  ApplicationStatus,
+  InstitutionLocation,
+} from "@sims/sims-db";
+import { FieldSortOrder } from "@sims/utilities";
+import { getUserFullName } from "../../../../utilities";
+
+describe("ApplicationOfferingChangeRequestInstitutionsController(e2e)-getCompletedApplications", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let collegeFLocation: InstitutionLocation;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    // College F.
+    const { institution: collegeF } = await getAuthRelatedEntities(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    collegeFLocation = createFakeInstitutionLocation(collegeF);
+    await authorizeUserTokenForLocation(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+      collegeFLocation,
+    );
+  });
+
+  it(`Should return all completed application offering change requests for the location in ${FieldSortOrder.ASC} order of name when requested.`, async () => {
+    // Arrange
+
+    // Student 1 has a completed application to the institution.
+    await saveFakeApplicationDisbursements(
+      db.dataSource,
+      { institutionLocation: collegeFLocation },
+      {
+        applicationStatus: ApplicationStatus.Completed,
+      },
+    );
+    // Student 2 has an in-progress with student application offering a change request for the institution.
+    await saveFakeApplicationOfferingRequestChange(db.dataSource, {
+      institutionLocation: collegeFLocation,
+    });
+    // Student 3 has an approved application offering change request for the institution.
+    const applicationOfferingChange3 =
+      await saveFakeApplicationOfferingRequestChange(
+        db.dataSource,
+        {
+          institutionLocation: collegeFLocation,
+        },
+        {
+          applicationOfferingChangeRequestStatus:
+            ApplicationOfferingChangeRequestStatus.Approved,
+        },
+      );
+    // Student 4 has an in progress with student application offering change request with a different institution.
+    await saveFakeApplicationOfferingRequestChange(db.dataSource);
+    // Student 5 has a declined by student application for the institution, that have an approved application offering change request.
+    const applicationOfferingChange5 =
+      await saveFakeApplicationOfferingRequestChange(
+        db.dataSource,
+        {
+          institutionLocation: collegeFLocation,
+        },
+        {
+          applicationOfferingChangeRequestStatus:
+            ApplicationOfferingChangeRequestStatus.DeclinedByStudent,
+        },
+      );
+    // Student 6 has a declined by student application for the institution, that have an approved application offering change request.
```

Comment:
`declined by ministry`?

Reply:
please refresh the Git,its already addressed